src/models/nebula36-1/assembly.md

```diff
@@ -0,0 +1,399 @@
+2. Unscrew and remove the 2.5" drive cage's cover.
+
+![2.5" SATA drive cage cover](./img/25-drive-cover.webp)
+
+3. Unscrew and remove the 2.5" drive cage.
+
+![2.5" SATA drive cage](./img/25-drive-cage-screws.webp)
+
+4. If the SATA backplane's screw holes are covered by a translucent film, remove the film.
+5. Place the SATA backplane into the drive cage.
+    - The backplane should be oriented so the power plug will be at the top of the board after installing the drive cage back into the chassis.
+6. Screw the backplane into the drive cage from the outside.
+
+![2.5" SATA backplane screws](./img/sata-backplane-screws.webp)
+
+7. Screw the drive cage back onto the chassis.
+8. Connect the `POWER0` header on the back of the SATA backplane to the power supply, and the `DATA0` and `DATA1` ports to two of the motherboard's SATA ports.
+
+![2.5" SATA backplane headers](./img/sata-backplane-headers.webp)
+
+## Installing 2.5" drives:
+
+The drive cage can hold two 2.5" drives. Drives of 7.5mm, 9mm, or 15mm are supported. Airflow will decrease below recommended levels with drives larger than 15mm.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to install 2.5" drives:
+
+1. [Remove the top case](#removing-the-top-case).
+2. Unscrew and remove the 2.5" drive cage's cover.
+
+![2.5" SATA drive cage cover](./img/25-drive-cover.webp)
+
+3. Remove the plastic ring from the top crossbar and slide out four screws for each drive being installed.
+
+![2.5" SATA drive screws on top crossbar](./img/25-drive-screws.webp)
+
+4. Insert two screws into each side of the 2.5" drive, then slide the drive into the drive cage, aligning the rubber mounts on the screws around the metal of the drive cage.
+    - The larger portion of the SATA connector (the power section) should be at the top, while the smaller portion (the data section) should be at the bottom.
+5. If the [optional SATA backplane](#installing-the-sata-backplane) is not present, connect the SATA data and power cables from the back of the drive cage.
+    - A cutout in the chassis allows easy access to the back of the drive cage.
+
+## Replacing the bottom case fan:
+
+The 140mm bottom case fan (`BQ SIW4-14025-LF-PWM`) is preinstalled, but can be replaced if it becomes defective. The fan can also be removed to clean its dust filter.
+
+- **Tools required:** Cross-head (Phillips) screwdriver  
+
+### Steps to replace the bottom case fan:
+
+1. [Remove the top case](#removing-the-top-case).
+    - The [side GPU brace](#removing-the-gpu-brace) can optionally be removed to make working with the fan easier.
+2. If the system is already assembled, unplug the fan from the fan splitter board or motherboard.
+3. Set the machine down on its side and unscrew the four fan screws.
+
+![Bottom fan screws](./img/bottom-fan-screws.webp)
+
+4. When reinstalling the fan, the components should be in the following order:
+    - Chassis
+    - Dust filter
+    - Acrylic spacer
```

Comment:
I believe this filter also does not require an acrylic spacer

Reply:
Addressed.